### ISSUE TITLE
Unsupported platform: {"os":"linux","arch":"arm64"}

### ISSUE THREAD

**Issue by abacaj** [giggio/node-chromedriver#173]
Unable to install on my linux box running `Ubuntu Xenial (16.04 LTS) (GNU/Linux 4.4.127-mainline-rev1 aarch64)`:

```
1262 warn optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
1263 warn notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})
1264 verbose notsup SKIPPING OPTIONAL DEPENDENCY: Valid OS:    darwin
1264 verbose notsup SKIPPING OPTIONAL DEPENDENCY: Valid Arch:  any
1264 verbose notsup SKIPPING OPTIONAL DEPENDENCY: Actual OS:   linux
1264 verbose notsup SKIPPING OPTIONAL DEPENDENCY: Actual Arch: arm64
1265 verbose stack Error: chromedriver@2.42.0 install: `node install.js`
1265 verbose stack Exit status 1
1265 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
1265 verbose stack     at EventEmitter.emit (events.js:182:13)
1265 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
1265 verbose stack     at ChildProcess.emit (events.js:182:13)
1265 verbose stack     at maybeClose (internal/child_process.js:962:16)
1265 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
1266 verbose pkgid chromedriver@2.42.0
1267 verbose cwd /root/browserless
1268 verbose Linux 4.4.127-mainline-rev1
1269 verbose argv "/usr/bin/node" "/usr/bin/npm" "i" "--save" "chromedriver"
1270 verbose node v10.12.0
1271 verbose npm  v6.4.1
1272 error code ELIFECYCLE
1273 error errno 1
1274 error chromedriver@2.42.0 install: `node install.js`
1274 error Exit status 1
1275 error Failed at the chromedriver@2.42.0 install script.
1275 error This is probably not a problem with npm. There is likely additional logging output above.
1276 verbose exit [ 1, true ]
```

Any ideas?

**Comment by abacaj** [comment]
Running from source:

`Only Linux 64 bits supported.`

My platform is 64 bit: 
`uname -a`
`Linux sys-one 4.4.127-mainline-rev1 #1 SMP Sun Apr 8 10:44:07 UTC 2018 aarch64 aarch64 aarch64 GNU/Linux`

**Comment by giggio** [comment]
We need to understand what node is reporting. Can you try running `node -e 'console.log(process.arch)'` and post the output?

As you can see on the logic here:
https://github.com/giggio/node-chromedriver/blob/9439b26a787a2d5092bc1cf84043adc5cbe4658a/install.js#L24

That is what we are checking.

**Comment by giggio** [comment]
It seems, from your output, that you are running arm: `Actual Arch: arm64`.
Please confirm if that is the case. If it is, Chromedriver does not support ARM, AFAIK. It is weird it even runs, as the only compilation available for Linux is for 64 bits:
https://chromedriver.storage.googleapis.com/index.html?path=2.42/

**Comment by giggio** [comment]
Closed by #175. Published as `2.42.1`.
